assets/js/modules/analytics/components/module/ModulePopularPagesWidget/index.js

```diff
@@ -83,17 +84,41 @@ export default function ModulePopularPagesWidget( {
 		limit: 10,
 	};
 
-	const report = useSelect( ( select ) =>
-		select( MODULES_ANALYTICS ).getReport( args )
-	);
-	const loaded = useSelect( ( select ) =>
-		select( MODULES_ANALYTICS ).hasFinishedResolution( 'getReport', [
-			args,
-		] )
-	);
-	const error = useSelect( ( select ) =>
-		select( MODULES_ANALYTICS ).getErrorForSelector( 'getReport', [ args ] )
-	);
+	const { report, titles, loaded, error } = useSelect( ( select ) => {
+		const data = {
+			report: select( MODULES_ANALYTICS ).getReport( args ),
+			loaded: false,
+			error: select( MODULES_ANALYTICS ).getErrorForSelector(
+				'getReport',
+				[ args ]
+			),
+			titles: undefined,
+		};
+		const reportLoaded = select(
+			MODULES_ANALYTICS
+		).hasFinishedResolution( 'getReport', [ args ] );
+		let hasLoadedPageTitles = true;
```

Comment:
And here, `false` by default?

Reply:
https://github.com/google/site-kit-wp/pull/3958/commits/3cddcca994d8eb6a2e939ac3aca4998081ba16f2